src/pug/components/input/input.pug

```diff
@@ -12,3 +12,17 @@ mixin checkbox(data)
   .checkbox 
     input(type="checkbox")
     span= data
+
+mixin select(name,select,options)
+  .select-option
+    label(for="sort").mr-2=name
+      |:
+    select(name="sort")
```

Comment:
for thì sẽ đi kèm với ID. Vì này nó giống như component nên đừng set cứng for, name cho nó nhé
Còn name là name chứ, sao lấy name đặt cho label
```suggestion
    label(for="sort").mr-2=label
      |:
    select(name="sort")
```